app/controllers/concerns/unconfirmed_user_concern.rb

```diff
@@ -46,7 +46,7 @@ def stop_if_invalid_token
 
   def email_confirmation_token_validator
     @email_confirmation_token_validator ||= EmailConfirmationTokenValidator.
-                                            new(@email_address, current_user)
+      new(@email_address, current_user)
```

Comment:
maybe we use the `begin` block syntax here

```
    @email_confirmation_token_validator ||= begin
      EmailConfirmationTokenValidator.new(@email_address, current_user)
    end
```

Reply:
I don't know that I've seen this usage before. I love this, though.